src/science/canopy/cbl_init_wetfac_mod.f90

```diff
@@ -0,0 +1,103 @@
+MODULE cable_init_wetfac_mod
+   
+    IMPLICIT NONE
+
+    CONTAINS
+
+    SUBROUTINE initialize_wetfac( &
+        mp, &
+        ssnow_wetfac, &
+        soil_swilt, &
+        soil_sfc, & 
+        ssnow_wb, &
+        ssnow_wbice, &
+        ssnow_snowd, &
+        veg_iveg, &
+        met_tk, &
+        Ctfrz &
+    )
+
+        !! Initialize the surface wetness factor of the soil/snow (ssnow_wetfac) array
```

Comment:
Can we upgrade this to the min. requirements for in-code documentation: https://cable.readthedocs.io/en/latest/developer_guide/documentation_guidelines/science_doc/? 

You won't be able to detail much more but can use warnings to highlight the lack of provenance of the science and at least put the formatting in place.
Just realising the guidelines do not mention the copyright, I'll put that to add to the guidelines. Need to add copyright statement at the top of each file. Some files have it...